workos/organizations.py

```diff
@@ -223,6 +224,14 @@ def delete_organization(self, organization_id: str) -> None:
             method=REQUEST_METHOD_DELETE,
         )
 
+    def list_organization_roles(self, organization_id: str) -> RolesList:
```

Comment:
How do we feel about this being housed in the organizations module? It's a subresource in the API, but should be separated as such? Alternatively we could introduce the idea of a sub-module.

Reply:
See discussion in Slack.